client-src/elements/chromedash-admin-blink-page.ts

```diff
@@ -191,8 +186,8 @@ export class ChromedashAdminBlinkPage extends LitElement {
     if (location.hash) {
       const hash = decodeURIComponent(location.hash);
       if (hash) {
-        const el = this.shadowRoot.querySelector(hash);
-        el.scrollIntoView(true, {behavior: 'smooth'});
+        const el = this.shadowRoot!.querySelector(hash);
+        el!.scrollIntoView({behavior: 'smooth'});
```

Comment:
This could be a behavior change, but it's probably the intended behavior.

Reply:
yeah it only accepts 1 argument, and should be the configs